packages/shared/components/modals/ExpirationTimePickerMenu.svelte

```diff
@@ -37,27 +43,26 @@
 
     $: {
         switch (selected) {
-            case 'none':
-                value = null
-                break
-            case '1hour':
+            case ExpirationTime.OneHour:
                 value = dateIn1Hour
                 break
-            case '1day':
+            case ExpirationTime.OneDay:
                 value = dateIn1Day
                 break
-            case '1week':
+            case ExpirationTime.OneWeek:
                 value = dateIn1Week
                 break
-            case 'custom':
+            case ExpirationTime.None:
+                value = null
                 break
+            case ExpirationTime.Custom:
             default:
-                value = null
```

Comment:
Should we not have the `value = null` statement for the `default` case?

Reply:
In theory we never have the `default` case. But if we have it, its better to keep the current value instead of resetting it